packages/core/util/FileSystem.ts

```diff
@@ -12,9 +12,9 @@ export class FsFileSystem implements IFileSystem {
 			return false;
 		}
 	}
-	public readFile(filePath: string, encoding?: string): string {
+	public readFile(filePath: string, encoding?: BufferEncoding): string {
```

Comment:
Uf, this should... also reflect in the interface? Since it's quite useless in the actual implementation that nothing should refer directly.

Should we consider just defaulting to UTF8 in general?